pkg/controller/namespace/namespace_controller.go

```diff
@@ -163,3 +177,16 @@ func namespaceInSlice(namespace string, safeList []string) bool {
 	}
 	return false
 }
+
+// check if rolebinding is already created in the namespace
+func rolebindingInNamespace(rolebinding *v1.RoleBinding, roleBindingList *v1.RoleBindingList) bool {
+	list := roleBindingList.Items
+	roleBindingName := rolebinding.Name
+
+	for _, rb := range list {
+		if rb.Name == roleBindingName {
```

Comment:
Just thought of a potential issue (not sure we can solve it here). What if a permission gets updated? Then the role binding will also need to be updated. But if the names are the same, the new permissions won't be applied. I don't think DeepEqual would work here though which would be the easiest fix.